extensions/ql-vscode/test/unit-tests/common/files.test.ts

```diff
@@ -417,3 +459,41 @@ describe("walkDirectory", () => {
     expect(files.sort()).toEqual([file1, file2, file3, file4, file5, file6]);
   });
 });
+
+describe("findCommonParentDir", () => {
```

Comment:
The docs on `findCommonParentDir` says it works with absolute and relative paths, but the unit tests only cover relative paths.

Could also do with more unit tests in general, if that's ok. There are so many edge cases with paths 😭

Reply:
I've added a few but please let me know if you can think of any more test cases.